client_dashboard/common/src/commonMain/kotlin/org/thechance/common/data/remote/gateway/taxi_gateway/ITaxiGateway.kt

```diff
@@ -0,0 +1,11 @@
+package org.thechance.common.data.remote.gateway.taxi_gateway
+
+import org.thechance.common.domain.entity.AddTaxi
+import org.thechance.common.domain.entity.DataWrapper
+import org.thechance.common.domain.entity.Taxi
+
+interface ITaxiGateway {
+    suspend fun getTaxis(): DataWrapper<Taxi>
+    suspend fun createTaxi(taxi: AddTaxi): Taxi
```

Comment:
AddTaxi  data class name changed into NewTaxiInfo so taxi should be of NewTaxiInfo type and not AddTaxi

Reply:
just change addTaxi to NewTaxiInfo that  already exist

Reply:
![image](https://github.com/TheChance101/beep-beep/assets/105972942/b5ca6832-6ad1-47da-ba65-6295ee032501)

Reply:
resolved 👍